app/main.py

```diff
@@ -2,12 +2,11 @@ def copy_file(command: str) -> None:
     listed_command = command.split()
     first_file = listed_command[1]
     second_file = listed_command[2]
```

Comment:
You must check that len(listed_command) == 3 before this code because if len(listed_command) is 1 or 0, the code will be failed.